Kitodo-Docket/src/main/java/org/kitodo/docket/Docket.java

```diff
@@ -27,11 +27,12 @@ public File generateDocket(DocketData docketData, URI xslFileUri) throws IOExcep
         ExportDocket exportDocket = new ExportDocket();
 
         File file = new File("docket.pdf");
-        FileOutputStream fileOutputStream = new FileOutputStream(file);
 
-        exportDocket.startExport(docketData, fileOutputStream, new File(xslFileUri));
-        fileOutputStream.flush();
-        fileOutputStream.close();
+        try (FileOutputStream fileOutputStream = new FileOutputStream(file)) {
+            exportDocket.startExport(docketData, fileOutputStream, new File(xslFileUri));
+        } catch (IOException e) {
```

Comment:
This looks strange. You are catching an IOException, just to throw it again? Which CID do you want to fix here?

Reply:
Yes, it is more elegant way of ensure that fileOutputStream will be closed. We rethrown it as we have convention that all exceptions are going to be passed to Kitodo Core, not caught in modules.

Reply:
It would be even more elegant if you simply omitted the catch part. It is not needed to achieve the desired behaviour. See https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html. I suggest to fix that for all code locations.